classes/fields/date.php

```diff
@@ -111,6 +111,15 @@ public function options() {
 				),
 				'dependency' => true,
 			),
+			static::$type . '_year_range'      => array(
+				'label'   => __( 'Year range', 'pods' ),
+				'default' => '',
+				'type'    => 'text',
+				'help'    => sprintf(
```

Comment:
Can we show an example of at least 1-2 options in this help text? This isn't super helpful if we just have a link here.